code/modules/mob/living/carbon/human/human_life.dm

```diff
@@ -604,7 +604,7 @@
 
 	if(!check_death_method())
 		if(health <= HEALTH_THRESHOLD_DEAD)
-			var/deathchance = min(99, ((getBrainLoss() * -5) + (health + (getOxyLoss() / 2))) * -0.01)
+			var/deathchance = min(99, ((getBrainLoss() / 2) + (health + (getOxyLoss() / -2))) * -0.1)
```

Comment:
The 0.1 being negative isn't correct I think. Currently, more oxyloss reduces the chance of dying again. 

Also, I think this would work better if we used clamp to put it between 1 and 100, as an extra precaution.

Reply:
As long as getOxyLoss is divided by a negative number, it correctly contributes to deathchance.

Reply:
Order of operations my beloathed.